app/res/values/strings.xml

```diff
@@ -49,7 +49,7 @@
     <string name="privilege_not_enabled_text">Special privileges are not enabled on your device.</string>
     <string name="privilege_claim_failed">Could not claim privileges</string>
     <string name="privilege_instructions">If you have a superuser account on HQ, you can enable certain privileges on your device by going to the \'Enable Privileges on Mobile\' tab of your \'My Account Settings\' page, and then scanning the barcode that appears there.</string>
-
+    <string name="see_available_apps">See Available Apps</string>
```

Comment:
is the reason for having this here instead of in `android_startup_strings` that you want to reference this directly from a layout XML file?